packages/manager/modules/sms/src/sms/batches/history/delete/delete.routing.js

```diff
@@ -0,0 +1,21 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('sms.service.batches.history.delete', {
+    url: '/delete?batchId&batchName&serviceName',
```

Comment:
```suggestion
    url: '/delete?batchId&batchName',
```

Reply:
I think serviceName is already in $transition$

Reply:
Done